web/modules/custom/hoeringsportal_content_access/hoeringsportal_content_access.module

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * @file
+ * Module for node access.
+ */
+
+use Drupal\Core\Access\AccessResultForbidden;
+use Drupal\Core\Access\AccessResultNeutral;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\node\NodeInterface;
+
+/**
+ * Implements hook_node_access().
+ */
+function hoeringsportal_content_access_node_access(NodeInterface $node, $operation, AccountInterface $account) {
+  $bundle = $node->bundle();
+  if ($bundle == 'project' && $operation == 'view') {
+    $access = new AccessResultForbidden();
+  }
+  else {
+    $access = new AccessResultNeutral();
+  }
+  return $access;
```

Comment:
This should be `return ($bundle == 'project' && $operation == 'view') ? new AccessResultForbidden() : new AccessResultNeutral();`.

Reply:
Changed return